src/compile/frontends/wasm.ts

```diff
@@ -0,0 +1,71 @@
+import { gte, satisfies } from "semver";
+import { assert } from "../../misc";
+import { ImportFinder } from "../utils";
+import { Compiler } from "./base";
+
+export abstract class WasmCompiler extends Compiler {
+    public readonly moduleName: string;
+    public readonly module: any;
+
+    constructor(version: string) {
+        super(version);
+        this.moduleName = `solc-${this.version}`;
+        this.module = require(this.moduleName);
```

Comment:
Maybe it would be better to not do this in constructor. A getter maybe an alternative to not store entire Wasm reference in memory. It would also be used not regularly. NodeJS is caching `require()`d modules, as I recall.

Reply:
Fixed in https://github.com/ConsenSys/solc-typed-ast/pull/88. Now is done in getter